buyer-app/buyer-app/src/state/entities/closeDataOrder/sagas.js

```diff
@@ -14,22 +14,22 @@ function* closeDataOrder(action) {
 
   try {
     yield call(DataOrdersHelpers.closeOrder, dataOrder.orderAddress);
-
+    
     yield put(Actions.closeDataOrderSucceed({ dataOrder }));
-
+    
     yield put(
       NotificationActions.createNotification({
         message:
-          "The data order is closed. Data sellers will receive theis payment.",
+        "The data order is closed. Data sellers will receive their payments.",
```

Comment:
```suggestion
        "The data order has started to close.",
```
Maybe by saying this it's clearer that is not an instant action